Makefile

```
@@ -28,12 +28,12 @@ build:
 	PKG_NAME=rust-api-server
 	BUILD_VERSION=$(shell git describe --long)
 	BUILD_RELEASE=$(shell git describe --tags --abbrev=0)
-	BUILDKIT_PROGRESS=plain
-	DOCKER_BUILDKIT=1
-	docker build --ssh default -t $(PKG_NAME):$(BUILD_VERSION) --target=prod .
+	BUILDKIT_PROGRESS=plain	docker build --ssh default -t $(PKG_NAME):$(BUILD_VERSION) --target=public_server_prod .
```

```suggestion
	BUILDKIT_PROGRESS=plain	docker build -t $(PKG_NAME):$(BUILD_VERSION) --target=public_server_prod .
```
remove it in all places